### PR TITLE
Approved 404 message

### PR DESCRIPTION
**Pull request by shaedrich**
Fixes #548

### CODE REVIEW

**Comment by KamasamaK** [comment]
- Quotes are incorrectly escaped as `\"` instead of `""` and should not at all be escaped around `url.name` since you actually want that string to end and start there, respectively. Escapes are missing for the second `href`.
- This ironically has a spelling mistake in "spelling misstake".
- Some of the other wording and grammar could be improved. For example, "Please make sure, you did not misspelled it" has a misplaced comma and should be "misspell".

**Comment by shaedrich** [comment]
Isn't that ironic?

Well, that seems to be rather embarrassing...

File: doc.cfm [hunk 1]
```diff
@@ -31,7 +31,12 @@
 			<cfset ArrayAppend(possible, i)>
 		</cfif>
 	</cfloop>
-	<cfset data = {name = url.name, description="Sorry we don't have any docs matching that name. If we should have a doc for this, please send us a pull request.", type="404", related=possible}>
+	<cfset data = {
+		name = url.name,
+		description = "Sorry we don't have any docs matching that name. If we should have a doc for this, please send us a <a href=""https://github.com/foundeo/cfdocs/edit/master/data/en/" & url.name & ".json"">pull request</a> on github.\n\nYou can easily access functions and tags using an url like <a href=""http://cfdocs.org/hash"">cfdocs.org/hash</a>. Just hit <code>/tag-name</code> or <code>/function-name</code> or use the search box above.\n\nMaybe it's just a spelling mistake. Please make sure you did not misspell it before you open a pull request or issue!",
```
Comment (pfreitag) [inline doc.cfm]: The link will always resolve to a 404: https://github.com/foundeo/cfdocs/edit/master/data/en/foo.json

File: doc.cfm [hunk 2]
```diff
@@ -31,7 +31,12 @@
 			<cfset ArrayAppend(possible, i)>
 		</cfif>
 	</cfloop>
-	<cfset data = {name = url.name, description="Sorry we don't have any docs matching that name. If we should have a doc for this, please send us a pull request.", type="404", related=possible}>
+	<cfset data = {
+		name = url.name,
+		description = "Sorry we don't have any docs matching that name. If we should have a doc for this, please send us a <a href=""https://github.com/foundeo/cfdocs/new/master/data/en?filename=" & url.name & ".json"">pull request</a> on github.\n\nYou can easily access functions and tags using an url like <a href=""http://cfdocs.org/hash"">cfdocs.org/hash</a>. Just hit <code>/tag-name</code> or <code>/function-name</code> or use the search box above.\n\nMaybe it's just a spelling mistake. Please make sure you did not misspell it before you open a pull request or issue!",
```
Comment (pfreitag) [inline doc.cfm]: please use `encodeForURL` on url.name

File: doc.cfm [hunk 3]
```diff
@@ -32,7 +32,7 @@
 		</cfif>
 	</cfloop>
 	<cfset data = {
-		name = url.name,
+		name = encodeForURL(url.name),
 		description = "Sorry we don't have any docs matching that name. If we should have a doc for this, please send us a <a href=""https://github.com/foundeo/cfdocs/new/master/data/en?filename=" & url.name & ".json"">pull request</a> on github.\n\nYou can easily access functions and tags using an url like <a href=""http://cfdocs.org/hash"">cfdocs.org/hash</a>. Just hit <code>/tag-name</code> or <code>/function-name</code> or use the search box above.\n\nMaybe it's just a spelling mistake. Please make sure you did not misspell it before you open a pull request or issue!",
```
Comment (pfreitag) [inline doc.cfm]: Sorry should have been more clear... In the description you have a link to github with url.name in the querystring, that is where it should have `encodeForURL`

**Comment by pfreitag** [comment]
thanks

**Comment by shaedrich** [comment]
You're welcome

**Comment by shaedrich** [comment]
There is still something wrong with it: #683

**Pull request merged**